wikiclue/src/routes/manage-friends/+page.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+    import HeaderBar from "../../components/headerBar.svelte";
+    import Overlay from '../../components/overlay.svelte';
+    import { writable } from 'svelte/store';
+
+    const friendRemoveOverlayVis = writable(false);
+
+    function removeFriend(){
+        //remove friend logic
+    }
+</script>
+
+<HeaderBar />
+<div class="manage-friends">
+    <h1>Manage Friends</h1>
+    <div class="friendboard-container">
+        <div class="friends-container">
+            <h3>View and manage your friends!</h3>
+            <div class ="users-container">
+                <p class="friend-name">LitLover911<button on:click={() =>{friendRemoveOverlayVis.set(true)}}>Remove</button></p>
+                <p class="friend-name">Chris_Morroco <button on:click={() => {friendRemoveOverlayVis.set(true)}}>Remove</button></p>
+                <p class="friend-name">Bonnie <button on:click={() => {friendRemoveOverlayVis.set(true)}}>Remove</button></p>
+                <p class="friend-name">Clyde <button on:click={() => {friendRemoveOverlayVis.set(true)}}>Remove</button> </p>
```

Comment:
Can we make the remove button bigger? Maybe a bit more padding as they look small on a larger screen
![Screenshot 2024-03-17 233832](https://github.com/jmcmurtry/wikiclue/assets/77893842/47e76940-3578-4590-8d34-47e37286268c)